.travis.yml

```diff
@@ -8,15 +9,20 @@ os:
   - linux
 env:
   global:
-    - TA_INCLUDE_PATH=$HOME/dependencies/include
-    - TA_LIBRARY_PATH=$HOME/dependencies/lib
-    - LD_LIBRARY_PATH=$HOME/dependencies/lib
+    - DEPS_DIR=$HOME/dependencies
+  matrix:
+    - WITH_TA_LIBRARY=yes
+      TA_INCLUDE_PATH=$DEPS_DIR/include
+      LD_LIBRARY_PATH=$DEPS_DIR/lib
+      TA_LIBRARY_PATH=$DEPS_DIR/lib
```

Comment:
After the **Build ta-lib from source if the library is not found** (https://github.com/mrjbq7/ta-lib/pull/111#issuecomment-277419330) feature is merged, I'll add a line of 

```yaml
- WITH_TA_LIBRARY=no
```

in the matrix section to setup test cases for the feature.